tests/simpleKVBC/TesterReplica/setup.hpp

```diff
@@ -19,14 +19,25 @@
 #include "ICommunication.hpp"
 #include "Logger.hpp"
 #include "MetricsServer.hpp"
+#include "config/test_parameters.hpp"
 
-namespace concord {
-namespace kvbc {
+namespace concord::storage {
+class IDBClient;
+}
+
+namespace concord::kvbc {
+
+class IDataKeyGenerator;
 
 class TestSetup {
  public:
   static std::unique_ptr<TestSetup> ParseArgs(int argc, char** argv);
 
+  std::tuple<std::shared_ptr<concord::storage::IDBClient>,
+             std::shared_ptr<concord::storage::IDBClient>,
+             IDataKeyGenerator*>
+  get_dbadapter_configuration();
```

Comment:
[Minor]: I think a named struct would be clearer, especially with the same type (`IDBClient`) of the first two values.

Reply:
done